client/pages/SignUpPage.jsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Col, Input, Row } from "reactstrap";
+import { Head } from "../components";
+import countryData from "../utils/countries";
+import Select from "react-select";
+import React from "react";
+import "../public/styles/signupPage.scss";
+export default function() {
+  const options = [
+    { value: "student", label: "Student" },
+    { value: "teacher", label: "Teacher" },
```

Comment:
For now, could you remove the teacher and stakeholder option?

Reply:
this page is not whats being pushed. i'll delete it all together but the signup page is called register.jsx. This was from a while back.